test/edu-benefits/components/Address.unit.spec.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+import { assert, expect } from 'chai';
+
+import Address from
+    '../../../src/js/edu-benefits/components/Address';
+
+describe('<Address>', () => {
+  let component = null;
+
+  beforeEach(() => {
+    component = ReactTestUtils.renderIntoDocument(
+      <Address/>
+    );
+    assert.ok(component, 'Cannot even render component');
+  });
+
+  xit('has sane looking features', () => {
+    const inputs = ReactTestUtils.scryRenderedDOMComponentsWithTag(component, 'input');
+    expect(inputs).to.have.length(1);
```

Comment:
I think there are more than 1 input in the `Address` component (I know this test is commented out anyway, can we re-comment it as well?)